src/handlers/multiSig/multiSigCreated/handlers/addOrEditDomain.ts

```diff
@@ -0,0 +1,26 @@
+import { TransactionDescription } from 'ethers/lib/utils';
+import { ContractEvent, multiSigNameMapping } from '~types';
+import { createMultiSigInDB } from '../helpers';
+import { getPendingMetadataDatabaseId } from '~utils';
+
+export const handleAddOrEditDomainMultiSig = async (
+  colonyAddress: string,
+  event: ContractEvent,
+  parsedAction: TransactionDescription,
+): Promise<void> => {
+  if (!colonyAddress) {
+    return;
+  }
+
+  const { name } = parsedAction;
+
+  const pendingDomainMetadataId = getPendingMetadataDatabaseId(
+    colonyAddress,
+    event.transactionHash,
+  );
+
+  await createMultiSigInDB(colonyAddress, event, {
+    type: multiSigNameMapping[name],
+    pendingDomainMetadataId,
```

Comment:
We might need to add this to show the `CreatedIn` field on the completed action screen:

```ts
const { name, args } = parsedAction;
const [, , nativeDomainId] = args;

...

fromDomainId: getDomainDatabaseId(colonyAddress, nativeDomainId),
```

(I could be wrong, but it is there for the `editDomain` reputation motion)

Reply:
Hey @iamsamgibbs ! At first I tried to do that before I realised that Kerry has already "pre-added" this inside the `multiSigCreated/helpers.ts` file

![Screenshot 2024-07-03 at 14 37 44](https://github.com/JoinColony/block-ingestor/assets/50642296/ab541239-ea57-48af-998f-1be8d9faabf8)